intro-dplyr.Rmd

```diff
@@ -0,0 +1,343 @@
+---
+title: "Work with and manage your data efficiently in R: dplyr and tidyr"
+params:
+    datetime: "2018-10-26"
+    level: "Beginner" # or intermediate or advanced
+    videolink: "" # Keep empty if no link yet.
+    instructor: "Elena Dudukina"
+---
+
+```{r setup, echo=FALSE}
+source("R/utils.R")
+```
+
+## Session details
+
+- **Date of session**: `r format_date(params$datetime)`
+- **Instructor**: `r params$instructor`
+- **Session level**: `r params$level`
+    - **Part of the ["Beginner Series"](index.html)**
+- **Required packages to install**:
+
+    ```{r, eval=FALSE}
+    install.packages(c("dplyr", "tidyr"))
+    ```
+
+
+### Objectives
+
+1. To learn the difference between "messy" and "tidy" data and see the advantages of using data that are consistently structured.
+2. To get to know the tools that help tidying up and reshaping the data.
+3. To perform simple data transformations and create summaries.
+    
+**At the end of this session you will be able:**
+
+1. To perform simple data transformations using dplyr function `mutate()`.
+1. To select variables and observations to work with using dplyr functions `filter()`and `select()`.
+1. To order data by variable using dplyr function `arrange()`.
+1. To provide a simple data summary using dplyr functions `group_by()` and `summarise()`.
+1. To change the structure of the data using tidyr functions `gather()`, `separate()`, and `spread()`.
+
+## Resources for learning and help
+
+**For learning**:
+
+1. "R for Data Science" book: [Chapter on tidy data](http://r4ds.had.co.nz/tidy-data.html)
+2. [tidyr vignette](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html)
+3. [dplyr vignette](https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html)
+4. DataCamp course ["Working with Data in the Tidyverse"](https://www.datacamp.com/courses/working-with-data-in-the-tidyverse)
+    - Chapter 1 "Explore your data" is free (the following chapters are available after purchasing the DataCamp subscription)
+
+**For help**:
+    
+1. [Data wrangling cheatsheet](https://www.rstudio.com/wp-content/uploads/2015/02/data-wrangling-cheatsheet.pdf)
+2. [StackOverflow for tidyr](https://stackoverflow.com/questions/tagged/tidyr)
+3. [StackOverflow for dplyr](https://stackoverflow.com/questions/tagged/dplyr)
+4. Quick package information using RStudio: `?tidyr`, `?dplyr`
+
+## Quickly look over the data we will use
```

Comment:
I added this section. It's always good to quickly go over what the data looks like.